src/app/containers/Article/__snapshots__/index.test.jsx.snap

```diff
@@ -2,19 +2,23 @@
 
 exports[`ArticleContainer Component Composing the Article Container using the page handlers should compose articleContainer with the Page Handler in the correct order 1`] = `
 <div
-  id="PageWrapperContainer"
```

Comment:
So much nesting! A good case for ditching HOCs for hooks or render prop components.

Reply:
I'll be happy to do this in another PR. I'd rather this is kept within scope.

Reply:
I've created [this issue](https://github.com/bbc/simorgh/issues/4640) to address this.